src/main/java/core/basesyntax/service/impl/FileCsvWriterImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileCsvWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+public class FileCsvWriterImpl implements FileCsvWriter {
+    @Override
+    public void writeToFile(String content, String filePath) {
+        try {
+            Files.write(Path.of(filePath), content.getBytes());

```

Comment:
if u want, u can write string, and u don't need use method getBytes()
it will be: 'Files.writeString(Path.of(filePath), content);'